sdk/python/kfp/v2/google/client/client.py

```diff
@@ -167,6 +167,10 @@ def __init__(
       endpoint: AIPlatformPipelines service endpoint. Defaults to
         'us-central1-aiplatform.googleapis.com'.
     """
+    logging.warning(
```

Comment:
nit: use `FutureWarning`, which is not ignored by default. https://docs.python.org/3/library/warnings.html#warning-categories

e.g.:
https://github.com/kubeflow/pipelines/blob/78067d1dd72d14814146851eb476667ece5f88a9/sdk/python/kfp/dsl/_container_op.py#L1176

Reply:
Done :)